tests/enforcer/gce_firewall_enforcer_test.py

```diff
@@ -493,6 +569,14 @@ def test_missing_ip_protocol(self):
         with self.assertRaises(fe.InvalidFirewallRuleError):
             self.firewall_rules._check_rule_before_adding(self.test_rule)
 
+    def test_denied_missing_ip_protocol(self):
+      """A rule missing IPProtocol in an allow predicate raises an exception."""
```

Comment:
Nit, test comment should say 'in a denied predicate'

Reply:
Done